sensor_msgs/Image Image
string[] DetectedObjectList
string[] FullObjectList
re_msgs/DetectedObject[] Detections